### PR TITLE
Rethinking 2 - Chapter 5 Edits

### PR DESCRIPTION
**Pull request by yahrMason**
# Thank you for opening a pull request!

Please check our [style guide](https://docs.pymc.io/en/latest/contributing/jupyter_style.html), and also make sure that the notebooks you've modified pass the `pre-commit` checks. If, for example, you modified `notebook1.ipynb` and `notebook2.ipynb`, you could do this by running:

```bash
pre-commit run --files notebook1.ipynb notebook2.ipynb
```

You may need to run it a couple of times before all the checks pass.

### CODE REVIEW

**Comment by yahrMason** [comment]
@canyon289 from what I can tell, it looks like all of the notebooks under Rethinking2 have been converted to PyMC v4. As for your two questions, I certainly can change the env file. Is that as simple as just changing "pymc3" to "pymc"? And yes, I intend to convert all of them as I progress through the text. Should I open a single PR for this initiative and commit as I go along? I wasn't sure what the right approach was.

Thank you for your patience with me. Two of my goals this year are to learn more about Bayesian statistics and to contribute to open source. I thought this looked like a good exercise to get experience in both of those areas. I am still early in my learning journey.

**Comment by canyon289** [comment]
>  certainly can change the env file. Is that as simple as just changing "pymc3" to "pymc"?

Thanks so much! What I would suggest right now is creating a second `environment.yml` file labeled `environment_v4.yml` while were switching from v3 to v4. I would make it the versions of all the packages youre using right now, This will be more than just v4 as the aesara and aeppl versions will ned to be changed as well.

If you're going through the book checking the other notebooks will be a great way to learn as well.

As for this PR the precommit failed. Please run the isort and nblack commands to ensure all the formatting is correct. Thanks much for the PR!

**Comment by EAly** [comment]
@yahrMason  Thanks for the edits, it really helped.
I'm not sure, however, that all of the notebooks under Rethinking2 have been converted to PyMC v4.

File: .gitignore [hunk 1]
```diff
@@ -99,3 +99,4 @@ ENV/
 
 # mypy
 .mypy_cache/
+Rethinking_2/End_of_chapter_problems/Chapter_2_mine.ipynb
```
Comment (canyon289) [inline .gitignore:102]: Create a personal gitignore for this 

https://medium.com/@peter_graham/how-to-create-a-local-gitignore-1b19f083492b

File: Rethinking_2/environment_v4.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: stat-rethink2-pymc_v4
+channels:
+- conda-forge
+dependencies:
+  - jupyter
```
Comment (canyon289) [inline Rethinking_2/environment_v4.yml:5]: Please freeze the versions

File: .gitignore (hunk 1 above)
Comment (yahrMason) [reply .gitignore]: Fixed

File: Rethinking_2/environment_v4.yml (hunk 2 above)
Comment (yahrMason) [reply Rethinking_2/environment_v4.yml:5]: I believe this [commit ](https://github.com/pymc-devs/pymc-resources/pull/198/commits/8eaf23faadf24771522b092ab2e64f9b4eb5ab77)resolves this issue.

**Comment by canyon289** [comment]
Thank you!

**Pull request merged**